roles/nginxplus/files/conf/http/openpublishing.conf

```diff
@@ -4,6 +4,7 @@ proxy_cache_path /data/nginx/ojs-prod/NGINX_cache/ keys_zone=ojs-prodcache:10m;
 upstream ojs-prod1 {
     zone ojs-prod 64k;
     server ojs-prod1.princeton.edu:443;
+    #server ojs-prod2.princeton.edu:443;
```

Comment:
Should this be uncommented?

Reply:
no it shouldn't. Only one end point is active.